src/main/modules/oidc/index.ts

```diff
@@ -41,7 +41,10 @@ export class OidcMiddleware {
 
     server.get('/logout', function(req: Request, res){
       req.session.user = undefined;
-      req.session.save(() => res.redirect('/'));
+      req.session.save(() => {
+        res.redirect('/');
+        res.locals.isLoggedIn = false;
```

Comment:
do you mean to set the session?

Reply:
na, i think the placement is wrong hence ill change it, but in new version i pushed, in my opinion every request is going to hit that callback so it will check it there.

`if (req.session.user) {
        res.locals.isLoggedIn = true;
        return next();
      }
      res.locals.isLoggedIn = false;
      res.redirect('/login');`

Reply:
otherwise we'll have to return if their authenticated in each render response. i think this way is cleaner.